pkgs/applications/audio/strawberry/default.nix

```diff
@@ -0,0 +1,45 @@
+{ mkDerivation, stdenv, lib, fetchFromGitHub, cmake, pkgconfig
```

Comment:
I've never seen mkDerivation as a function arg but I guess it makes sense

Reply:
Though this does mean `stdenv` has not been used anywhere

Reply:
See https://github.com/NixOS/nixpkgs/issues/65399.

stdenv is used in ```stdenv.isLinux```.

Reply:
of course, my bad

Reply:
Thanks for the link as well